components/text-area/src/text-area/text-area.styles.js

```diff
@@ -29,7 +29,8 @@ export const styles = css`
 
     textarea:focus {
         outline: none;
-        box-shadow: 0 0 0 3px ${theme.focus};
+        box-shadow: inset 0 0 0 2px ${theme.focus};
```

Comment:
Should the spread radius here (the 2px) not be 3px?

Reply:
I think this is correct, he's using an inset box-shadow with a width of 2px plus a blue border of 1px, so the total visual width is 3px.

Reply:
That's right, the `box-shadow` and `border` combine to make 3px.

Reply:
Ah nice. Yeah I assumed it was intentional since it was changed from 3 to 2, but it stood out to me so I thought I'd ask.